sidecar/app/routes/start.py

```diff
@@ -87,6 +90,38 @@ def get_ipa_helper_status(
     return {"status": query.status.name}
 
 
+@router.get("/{query_id}/log-file")
+def get_ipa_helper_log_file(
```

Comment:
I would recommend to guarantee ordering on the server side. JS operates in my browser and it is busy with other things, so we should delegate hard work to server side. Besides that, it also makes API consistent

Reply:
btw can you point me to the file where traces are written out of order? I'd like to take a look and see if we may have an issue on the MPC side

Reply:
oh I don't think they should ever be out of order on the server side. they are only out of order when merging the 4 files together. for example a log from Helper 1 might arrive after a log from Helper 2, but have an earlier timestamp. 

this is particularly true when we're refreshing later and just consuming the file in it's entirety. 

let me point you to where the ordering happens. It should be pretty efficient.